prisma/schema.prisma

```diff
@@ -34,3 +45,10 @@ model ExerciseLink {
   exercise   Exercise @relation(fields: [exerciseId], references: [id], onDelete: Cascade)
   exerciseId Int
 }
+
+model TrainingPreference {
+  id        Int          @id @default(autoincrement())
+  type      TrainingType
+  time      Int
+  intensity String
+}
```

Comment:
Don't we need the relation of the firebase user or the userInfo?

Reply:
yes, i was thinking adding the relation once we create the firebaseUser scheme, but i will add the schema here